lib/test/test_script_constants.rb

```diff
@@ -62,18 +62,19 @@ def test_category_index
   end
 
   def test_category_priority
-    assert_equal 7, ScriptConstants.category_priority(:csf_international)
-    assert_equal 9, ScriptConstants.category_priority(:research_studies)
+    assert_equal 8, ScriptConstants.category_priority(:csf_international)
+    assert_equal 10, ScriptConstants.category_priority(:research_studies)
   end
 
   def test_assignable_info
-    assert_equal 1, ScriptConstants.assignable_info({name: 'dance-2019'})[:position]
-    assert_equal 2, ScriptConstants.assignable_info({name: 'dance-extras-2019'})[:position]
-    assert_equal 3, ScriptConstants.assignable_info({name: 'oceans'})[:position]
-    assert_equal 4, ScriptConstants.assignable_info({name: 'aquatic'})[:position]
-    assert_equal 5, ScriptConstants.assignable_info({name: 'hero'})[:position]
-    assert_equal 6, ScriptConstants.assignable_info({name: 'mc'})[:position]
-    assert_equal 7, ScriptConstants.assignable_info({name: 'minecraft'})[:position]
+    dance_index = 6
+    assert_equal dance_index, ScriptConstants.assignable_info({name: 'dance-2019'})[:position]
```

Comment:
[not blocking for this PR] i know this test has been around for a while, but i'm not sure how useful it is. it does prevent us from reordering this list (although it only tests the positions of scripts within `CATEGORIES[:hoc]`), but it seems like a warning comment is sufficient, which we already have:

https://github.com/code-dot-org/code-dot-org/blob/8727f5e9117d37ca69fd4fab6159103eb7804fde/lib/cdo/script_constants.rb#L17-L20

and it seems to only fail like it does in this case where we're intentionally changing the positions to add new scripts. 

this test was added in https://github.com/code-dot-org/code-dot-org/pull/19407 to fix a real regression, but i wonder if the case we ran into is still relevant or if we've made improvements elsewhere to avoid that situation from happening again.

just some thoughts 🤷🏻‍♀️